resources/views/dashboard.blade.php

```diff
@@ -0,0 +1,121 @@
+@extends('layout.layout')
+
+@section('contentcss')
+    <link rel="stylesheet" href="{{ Vite::asset('resources/css/dashboardstyles.css') }}">
```

Comment:
It is considered a good practice to rely on page-specific styles as less as possible. It avoids code duplication and helps maintainability.